state-loader.md

```diff
@@ -106,47 +106,32 @@ module STATE-LOADER
 Here we perform pre-proccesing on account data which allows "pretty" specification of input.
 
 ```{.k}
-    rule <k> load "pre" : { (ACCTID:String) : ACCT }             => mkAcct #parseAddr(ACCTID) ~> load "account" : { ACCTID : ACCT }                         ... </k>
-    rule <k> load "account" : { ACCTID: { KEY : VALUE , REST } } => load "account" : { ACCTID : { KEY : VALUE } } ~> load "account" : { ACCTID : { REST } } ... </k> requires REST =/=K .JSONList
-
-    rule <k> load "account" : { ((ACCTID:String) => #parseAddr(ACCTID)) : ACCT }                                ... </k>
-    rule <k> load "account" : { (ACCT:Int) : { "balance" : ((VAL:String)         => #parseWord(VAL)) } }        ... </k>
-    rule <k> load "account" : { (ACCT:Int) : { "nonce"   : ((VAL:String)         => #parseWord(VAL)) } }        ... </k>
-    rule <k> load "account" : { (ACCT:Int) : { "code"    : ((CODE:String)        => #parseByteStack(CODE)) } }  ... </k>
-    rule <k> load "account" : { (ACCT:Int) : { "storage" : ({ STORAGE:JSONList } => #parseMap({ STORAGE })) } } ... </k>
-```
+    rule <k> load "pre" : { (ACCTID:String) : ACCT } => mkAcct #parseAddr(ACCTID) ~> loadAccount #parseAddr(ACCTID) ACCT ... </k>
+
+    syntax EthereumCommand ::= "loadAccount" Int JSON
+ // -------------------------------------------------
+    rule <k> loadAccount ACCT { KEY : VALUE , REST }
+          => loadAccount ACCT { KEY : VALUE }
+          ~> loadAccount ACCT { REST }
+          ...
+         </k>
+      requires REST =/=K .JSONList
 
-The individual fields of the accounts are dealt with here.
+    rule <k> loadAccount _ { .JSONList } => . ... </k>
 
-```{.k}
-    rule <k> load "account" : { ACCT : { "balance" : (BAL:Int) } } => . ... </k>
-         <account>
-           <acctID> ACCT </acctID>
-           <balance> _ => BAL </balance>
-           ...
-         </account>
-
-    rule <k> load "account" : { ACCT : { "code" : (CODE:ByteArray) } } => . ... </k>
-         <account>
-           <acctID> ACCT </acctID>
-           <code> _ => CODE </code>
-           ...
-         </account>
-
-    rule <k> load "account" : { ACCT : { "nonce" : (NONCE:Int) } } => . ... </k>
-         <account>
-           <acctID> ACCT </acctID>
-           <nonce> _ => NONCE </nonce>
-           ...
-         </account>
-
-    rule <k> load "account" : { ACCT : { "storage" : (STORAGE:Map) } } => . ... </k>
-         <account>
-           <acctID> ACCT </acctID>
-           <storage> _ => STORAGE </storage>
-           <origStorage> _ => STORAGE </origStorage>
-           ...
-         </account>
+    rule <k> loadAccount ACCT { "balance" : (BAL:Int) } => . ... </k>
+         <account> <acctID> ACCT </acctID> <balance> _ => BAL </balance> ... </account>
```

Comment:
Let's do it in one shot:

```
    rule <k> loadAccount ACCT { "balance" : BAL:Int , REST => REST } ... </k>
```

So that loading each value happens in one step.